### ISSUE TITLE
Read-only table

### ISSUE THREAD

**Issue by lorang92** [Altinn/app-frontend-react#94]
## Description

Present tabular data in a table that is not editable.

## Screenshots
![image](https://user-images.githubusercontent.com/10416981/169050977-65ce31d2-46dd-4ad0-b7a0-560c659ec5ca.png)

See epic Altinn/app-frontend-react#107 for further details.

## Features
- **WCAG Requirements** - The component must meet all relevant requirements from WCAG
- **Responsiveness** - The table should work on all screen sizes with horizontal scroll in first version.
- **Read-only table** - Present tabular data in a table that is not editable.
- **Summation row** - One or more rows that show the calculation based on other values in the table. Should be presented as read-only fields and can be a row or a column
- **Header cells** - Up to two tier headers. The first [table example here](https://www.w3.org/WAI/tutorials/tables/irregular/) is a good example of what we need to support
- **Prefilled rows** - Use a datasource to create the number of rows needed. You should also be able to set a value as the row header. (The column to the left)

## Acceptance criteria
- Read only table is possible to add to studio and connect to a data model
- Reusable components are extracted to the design system
- Features described above is supported

## Specification tasks

- [x] Development tasks are defined
- [ ] Reusable components are identified and extracted

## Development tasks

- [ ] Studio changes
- [ ] App-frontend
- [ ] Design system

## Definition of done

- [ ] Documentation (docs.altinn.studio) is updated (if relevant)
  - [ ] Technical documentation
  - [ ] User documentation
- [ ] QA
- [ ] Manual test is complete (if relevant)
- [ ] Automated test is implemented (if relevant)
- [ ] All tasks in this userstory are closed (i.e. remaining tasks are moved to other user stories or marked obsolete)

**Comment by olemartinorg** [comment]
Closing, as this component will be implemented (albeit a bit more limited) as [List](https://github.com/Altinn/app-frontend-react/issues/578). That component won't be tied to the data model.
